src/torchcodec/decoders/_core/VideoDecoderOps.cpp

```diff
@@ -247,60 +247,59 @@ OpsFrameOutput get_next_frame(at::Tensor& decoder) {
 
 OpsFrameOutput get_frame_at_pts(at::Tensor& decoder, double seconds) {
   auto videoDecoder = unwrapTensorToGetDecoder(decoder);
-  auto result = videoDecoder->getFramePlayedAtNoDemux(seconds);
+  auto result = videoDecoder->getFramePlayedAt(seconds);
   return makeOpsFrameOutput(result);
 }
 
 OpsFrameOutput get_frame_at_index(
     at::Tensor& decoder,
-    int64_t stream_index,
+    [[maybe_unused]] int64_t stream_index,
```

Comment:
As mentioned in the PR description, I chose not to update the APIs of the ops and core APIs. Will deal with that in a follow-up.